src/nvim/event/process.c

```diff
@@ -257,8 +262,14 @@ static void children_kill_cb(uv_timer_t *handle)
     }
     uint64_t term_sent = UINT64_MAX == proc->stopped_time;
     if (kProcessTypePty != proc->type || term_sent) {
+#ifdef WIN32
+      proc->exit_signal = SIGKILL;
+#endif
       os_proc_tree_kill(proc->pid, SIGKILL);
     } else {
+#ifdef WIN32
+      proc->exit_signal = SIGTERM;
```

Comment:
even if it's not used in unix, we should still define this field and set it. Much better to avoid having different codepaths on different systems where possible, to the greatest extent possible.

Reply:
ok.  On UNIX we could then also have an assertion that we get the expected value from libuv.